src/client/Microsoft.Identity.Client/UI/IWebUIFactory.cs

```diff
@@ -15,6 +15,7 @@ internal interface IWebUIFactory
             RequestContext requestContext);
 
         bool IsSystemWebViewAvailable { get; }
-        
+        bool IsDesktopSession { get; }
```

Comment:
Would it be more accurate to name this `IsUiSession` since it's also used in Android and iOS factories?

Reply:
I will name it similarly to .NET5, i.e. https://github.com/dotnet/runtime/blob/4f67869210c4d1014255bd5ed907e78f795f9bef/src/libraries/System.Private.CoreLib/src/System/Environment.Windows.cs#L145-L165